.github/CODEOWNERS

```
@@ -58,3 +58,4 @@ i18n/en/docusaurus-plugin-content-docs/current/rock3/rock3a/os-config/rsetup.md
 static/img/common/etcher/rock5a-step3.webp @u7985
 docs/compute-module/nx5/radxa-os/apps-deploy/README.md @peterwang2050
 i18n/en/docusaurus-plugin-content-docs/current/nio/nio12l/download/README.md @peterwang2050
+.pre-commit-config.yaml @CodeChenL
```

```suggestion
.pre-commit-config.yaml @CodeChenL @RadxaYuntian 
```
然后移动到上面CI/CD section里面